src/amo/components/Addon/index.js

```diff
@@ -292,12 +293,17 @@ export class AddonBase extends React.Component {
     const addonPreviews = addon ? addon.previews : [];
 
     let isCompatible = false;
+    let isFeatured = false;
+    let isRestartRequired = false;
     let compatibility;
     if (addon) {
       compatibility = getClientCompatibility({
         addon, clientApp, userAgentInfo,
       });
       isCompatible = compatibility.compatible;
+      isFeatured = addon.is_featured;
+      // themes do not have this attribute
+      isRestartRequired = addon.is_restart_required || false;
```

Comment:
It would be nicer to just set this in the reducer rather than adding this check here. Would that be a huge pain?